toast/src/toast/node.rs

```diff
@@ -31,15 +33,52 @@ pub fn render_to_html(
     ];
     args.extend(filepaths.iter().cloned());
     let output = cmd("node", args).stderr_to_stdout();
-    run_cmd("sourceData", output, active_pb)?;
+    run_cmd("renderHTML", output, active_pb)?;
 
     Ok(())
 }
 
 #[instrument]
+#[cfg(windows)]
+pub fn render_to_html(
+    dir_of_input_files: String,
+    output_dir: String,
+    filepaths: Vec<String>,
+    npm_bin_dir: PathBuf,
+    toast_module_path: Option<PathBuf>,
+    active_pb: Arc<ProgressBar>,
+) -> Result<()> {
+    let bin = toast_module_path.unwrap().join("toast-render.mjs");
+    let bin_str = bin
+        .to_str()
+        .ok_or_else(|| eyre!("failed to make npm bin into str"))?;
+    let mut args: Vec<String> = vec![
+        "/c".to_owned(),
```

Comment:
why does this one only have `/c` as an argument and the other args in `sourceData` has `cmd`, `/c`?

Reply:
I kept the same structure as non-windows. See L70 where `cmd` is specified and passed these args.